examples/typescript/aws-move/main.ts

```diff
@@ -0,0 +1,283 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { Construct } from "constructs";
+//import { App, TerraformStack } from "cdktf";
+import { App, TerraformStack, TerraformIterator } from "cdktf";
+import { AwsProvider } from "./.gen/providers/aws/provider";
+import { S3Bucket } from "./.gen/providers/aws/s3-bucket";
+
+// UN-NESTING RESOURCE -> resource currently in nested construct, want to move it to higher layer
+
+// FIRST -> deploy, then comment out
```

Comment:
(just a note, no need to change anything)
An alternative to commenting out large chunks of code, could be to use an environment variable as a flag in order to change the behavior of the code. That way it's also programmatically accessible to tests that run synth + plan + apply in multiple phases.